testgrid/tgapi/pkg/persistence/pg.go

```diff
@@ -20,12 +20,6 @@ func MustGetPGSession() *sql.DB {
 		panic(err)
 	}
 
-	err = db.Ping()
```

Comment:
skaffold was not starting

Reply:
are you sure you should remove the db ping though? Doesn't that just run a `select 1` etc?

Reply:
we dont use this pattern in any other of our projects